recipes/aequilibrae/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "aequilibrae" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aequilibrae-{{ version }}.tar.gz
+  sha256: 64c30ae76460deccc197a732564d15f499b56a6c3d45ae720ca01978c18e841d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script_env:
+    - CFLAGS=-D_LIBCPP_DISABLE_AVAILABILITY  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - setuptools
+    - numpy
+    - cython
+    - pyaml
+    - pyqt
+    - requests
+    - scipy
+    - shapely
+    - pandas
+    - pyarrow
+    - pyproj
+    - wheel
+    - pip
```

Comment:
Only python deps which which
 - have C APIs which need to be linked against
 - are build tools
 
Should be listed in the host section. For example, numpy has a C API, if your package needs the numpy headers, then it should be listed in host. scipy has no C API AFAIK, so there is no reason for it to be listed in host. Cython is a build tool. requests is not a build tool or a package with C-API.

Reply:
Thanks for the comments, I believe I've fixed these. The CI failures are now pending a new AequilibraE release.